src/rogue/protocols/packetizer/Controller.cpp

```diff
@@ -55,8 +55,6 @@ rpp::Controller::Controller ( uint32_t segmentSize, rpp::TransportPtr tran, rpp:
 
    headSize_ = headSize;
    tailSize_ = tailSize;
-   enIbCrc_ = false; // Need to bring this up to the top level
```

Comment:
@slacrherbst: Where does the (enIbCrc_ = false) and (enObCrc_ = true) as default in the higher level software?